servicebox-app/src/main/resources/import.sql

```diff
@@ -1,6 +1,6 @@
 INSERT INTO claim(id, name, description, owner, serviceRequested, status, created) VALUES (1, 'mysql-demo',           'Quarkus JET Mysql demo',                 'quarkus team',               'mysql-7.5',        'new',      PARSEDATETIME('2022-10-18 10:30:00','yyyy-MM-dd HH:mm:ss'));
 INSERT INTO claim(id, name, description, owner, serviceRequested, status, created) VALUES (2, 'postgresql-team-dev',  'Spring Boot Config Server PostgreSQL',   'snowdrop team',              'postgresql-11.5',  'pending',  PARSEDATETIME('2022-10-18 08:22:00','yyyy-MM-dd HH:mm:ss'));
-INSERT INTO claim(id, name, description, owner, serviceRequested, status, created) VALUES (3, 'postgresql-team-test', 'Invoicing EAP Testing',                  'EAP QE team',                'postgresql-11.5',  'rejected', PARSEDATETIME('2022-10-18 11:00:00','yyyy-MM-dd HH:mm:ss'));
+INSERT INTO claim(id, name, description, owner, serviceRequested, status, created) VALUES (3, 'postgresql-team-test', 'Invoicing EAP Testing',                  'EAP QE team',                'postgresql-11.5',  'new',      PARSEDATETIME('2022-10-18 11:00:00','yyyy-MM-dd HH:mm:ss'));
 INSERT INTO claim(id, name, description, owner, serviceRequested, status, created) VALUES (4, 'mariadb-demo ',        'Dummy Java Maria DB ',                   'snowdrop r&d team',          'mariadb-7.5',      'bind',     PARSEDATETIME('2022-10-18 14:45:10','yyyy-MM-dd HH:mm:ss'));
 INSERT INTO claim(id, name, description, owner, serviceRequested, status, created) VALUES (6, 'postgresql-13 ',       'Quarkus Primaza & Postgresql',           'snwodrop & devtools teams',  'postgresql-13.5',  'bind',     PARSEDATETIME('2022-10-18 20:02:01','yyyy-MM-dd HH:mm:ss'));
 
```

Comment:
Shouldn't we to add something to populate a minima the ClaimingStatus table?

Reply:
I want to avoid populating this table here: the absence of claiming status is the same as there were no retries made. 
Anyway, I'm thinking to get rid of the ClaimingStatus table and having the attempts field directly into the Claim table. wdyt?

Reply:
Yes, I agree with having the attemps in the Claim table. I think at the moment it can do the job and would be more compact

Reply:
This is already done.